beater/sipcmbeat.go

```diff
@@ -326,20 +335,24 @@ func (bt *Sipcmbeat) publishEv(srcEv *calltr.EventData) {
 	}
 	addFields(event.Fields, "event.call_start", ed.StartTS)
 	addFields(event.Fields, "client.transport", ed.ProtoF.ProtoName())
-	if c, err := anonymization.EncryptIP(bt.encKey, ed.Src); err != nil {
-		addFields(event.Fields, "client.ip", c[:])
-	} else {
-		logp.Err("ERROR: client.ip encryption failed: %s \n", err)
-		stats.Inc(cntEvErr)
-		return
+	if bt.Config.UseIPAnonymization() {
+		if c, err := anonymization.EncryptIP(bt.encKey, ed.Src); err != nil {
+			addFields(event.Fields, "client.ip", c[:])
+		} else {
+			logp.Err("ERROR: client.ip encryption failed: %s \n", err)
+			stats.Inc(cntEvErr)
+			return
+		}
 	}
```

Comment:
what happens if no ip anonymization is used? Shouldn't it be:
```suggestion
	if bt.Config.UseIPAnonymization() {
		if c, err := anonymization.EncryptIP(bt.encKey, ed.Src); err != nil {
			addFields(event.Fields, "client.ip", c[:])
		} else {
			logp.Err("ERROR: client.ip encryption failed: %s \n", err)
			stats.Inc(cntEvErr)
			return
		}
	} else {
		addFields(event.Fields, "client.ip", ed.Src)        
	}
```
?